buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfig.kt

```diff
@@ -18,6 +19,9 @@ interface FladleConfig {
   @get:Optional
   val serviceAccountCredentials: RegularFileProperty
 
+  @get:Input
+  val flankVersion: Property<String>
```

Comment:
So far this looks good.
However this change to add the flankVersion to the FladleConfig might not actually work.

Can you verify in a test that we are able to override this version in different configs and it actually works?

That being said, i think this complicates things and I'm not sure its a feature we would like to have. Is there a way to achive this same functionality without exposing the version in this interface?

Reply:
I have a couple of ideas, thanks for the input!